src/dns.c

```diff
@@ -594,7 +632,7 @@ hsk_dns_qs_init(hsk_dns_qs_t *qs) {
   assert(qs);
 
   memset(qs->name, 0, sizeof(qs->name));
```

Comment:
sizeof will always be DNS_MAX_NAME

Reply:
fixed in f5200ab1251950b897e8ef3f71f5a11737aeaa5a